apps/src/dance/constants.ts

```diff
@@ -118,3 +118,10 @@ export const DANCE_FOREGROUND_EFFECTS = [
   'exploding_stars',
   'paint_drip',
 ];
+
```

Comment:
While you're in this file, could you remove the constants above ('COMMON_DANCE_BACKGROUND_EFFECT', 'EXTRA_AI_DANCE_BACKGROUND_EFFECTS', 'COMMON_DANCE_COLOR_PALETTES', 'COMMON_DANCE_COLOR_PALETTES', and 'DANCE_FOREGROUND_EFFECTS'? They are no longer needed being used after we switch from using OpenAI to the cached model.

Reply:
Done!